src/main/java/com/mycodefu/vsssfshss/http/HttpResourceManager.java

```diff
@@ -0,0 +1,34 @@
+package com.mycodefu.vsssfshss.http;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpResourceManager {
+    private static final Map<String, byte[]> resourceCache = new HashMap<>();
```

Comment:
Because the server is multithreaded, this HashMap should be synchronized. Perhaps use ConcurrentHashMap?